.github/workflows/test.yml

```diff
@@ -274,7 +260,7 @@ jobs:
         with:
           prefix-key: 'v3-rust-e2e'
           shared-key: ${{ matrix.e2e-type }}
-          cache-provider: 'buildjet'
+          cache-provider: 'github'
```

Comment:
do we not need to specify depot anywhere in here?

Reply:
nope! depot does a bunch of magic such that their cache/image builds etc work nicely with their runners

![image](https://github.com/user-attachments/assets/8b2a9ecc-1f3e-44a0-9912-7bee5189e7c1)